lib/BuildSystem/LaneBasedExecutionQueue.cpp

```diff
@@ -264,6 +264,7 @@ class LaneBasedExecutionQueue : public BuildExecutionQueue {
     int outputPipe[2]{ -1, -1 };
     if (shouldCaptureOutput) {
       if (::pipe(outputPipe) < 0) {
+        getDelegate().commandProcessStarted(context.job.getForCommand(), handle);
```

Comment:
Do you think my suggestion of changing the test code instead of the lib will work?